### DISCUSSION TITLE
Is there a way to toggle individual accordions without closing others.

### DISCUSSION THREAD

**Discussion by BranBayou** [saadeghi/daisyui#3197] [Q&A] [answered]
I'm currently working on a Vue.js project where I have multiple accordion components. Right now, when I open one accordion, any previously opened accordion automatically closes. I want to modify this behavior so that I can toggle each accordion individually, allowing multiple accordions to be open at the same time.

Is there a way to achieve this? If possible, please provide an example or guidance on how to implement this functionality. Thank you!

**Comment by saadeghi** [comment, marked as answer]
From docs:
https://daisyui.com/components/accordion/

>All radio inputs with the same name work together and only one of them can be open at a time. If you have more than one set of accordion items on a page, use different names for the radio inputs on each set.
